config/routes.rb

```diff
@@ -3,14 +3,16 @@
 
   ActiveAdmin.routes(self)
 
-  mount News::Engine => "/news"
+  scope "(:locale)", locale: /#{I18n.locale}/ do
+    mount News::Engine => "/"
+  end
 
   root 'main#index'
 
   scope module: 'home' do
     resources :user_sessions, only: [:new, :create]
     resources :users, only: [:new, :create]
-    get 'login' => 'user_sessions#new', :as => :login
+    get 'login' => 'user_sessions#new', as: :login
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.